src/main/java/uk/gov/hmcts/reform/propertiesvolume/PropertiesvolumeConfigProperties.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.propertiesvolume;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Configuration values for properties (e.g. secrets) loaded from a keyvault flexvolume.
+ *
+ */
+@ConfigurationProperties("spring.cloud.propertiesvolume")
+public class PropertiesvolumeConfigProperties {
```

Comment:
This looks weird to  me, how about?
```suggestion
public class PropertiesVolumeConfigProperties {
```

Reply:
I prefer to keep it consistent with the way the property itself is named.

Reply:
What do you meant by that? Its two words, shouldn't It be snakecased?